cs481-ekh/f21-welolo
# Task#4

Finished creating successful payment message. Right now it's just a div with a header inside of it, that is statically on the payment page. 

Verification Steps:
1. Run npm start in both the client and server folders
2. Open up your browser and navigate to 'localhost:3000'
3. Notice the message at the bottom of the page that says 'Payment Successful'
